### PR TITLE
Week 07 Synthesizing Assignments

### PR DESCRIPTION
**Pull request by saulatmajid**

### CODE REVIEW

**Comment by saulatmajid** [comment]
Hi Professor, My import has some problems. The checks are failing over and over because of that. Please suggest what to do.

**Comment by Shinseeker** [comment]
Hey Salatmajid. Check this link out. it may help. You can copy and paste it: http://learning.londonmet.ac.uk/computing/cc0002n/webpages/full_textChp6.htm

**Comment by cheuschober** [comment]
@saulatmajid: Note http://128.228.53.195/job/is210-2015-fall-01-07-synthesizing/8/console

When the build `timed out` it indicates you've created an infinite loop.

File: task_02.py [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    Args:
+        It takes 2 arguments, username and max_attempts, with 3 as default max.
+
+    Returns:
+        A boolean
+    """
+
+    authenticated = False
+    attempt = 0
+    while (attempt < max_attempts) or (not authenticated):
```
Comment (cheuschober) [inline task_02.py:22]: This is your infinite loop -- because you used an `or` if _either_ condition returned `True` it would continue.

**Pull request merged**